libnetwork/drivers/bridge/setup_ipv6_linux.go

```diff
@@ -39,14 +39,16 @@ func setupBridgeIPv6(config *networkConfiguration, i *bridgeInterface) error {
 	}
 
 	// Setting route to global IPv6 subnet
+	// TODO(robmry) - remove this? The bridge is 'down' at this point so I think it
+	//  always fails, and the route is added anyway when the bridge is set 'up'.
 	log.G(context.TODO()).Debugf("Adding route to IPv6 network %s via device %s", config.AddressIPv6.String(), config.BridgeName)
 	err = i.nlh.RouteAdd(&netlink.Route{
 		Scope:     netlink.SCOPE_UNIVERSE,
 		LinkIndex: i.Link.Attrs().Index,
 		Dst:       config.AddressIPv6,
 	})
 	if err != nil && !os.IsExist(err) {
-		log.G(context.TODO()).Errorf("Could not add route to IPv6 network %s via device %s: %s", config.AddressIPv6.String(), config.BridgeName, err)
+		log.G(context.TODO()).Debugf("Could not add route to IPv6 network %s via device %s: %s", config.AddressIPv6.String(), config.BridgeName, err)
 	}
```

Comment:
I would feel better if this was a more targeted suppression, just in case there's some convoluted code path which leads to this function while the bridge is up. `"network is down"` corresponds to errno `ENETDOWN` so it would be trivial to test for.
```go
if err != nil && !os.IsExist(err) {
    if errors.Is(err, syscall.ENETDOWN) {
        log.G(context.TODO()).Debugf("Could not add route to IPv6 network %s via device %s: %s", config.AddressIPv6.String(), config.BridgeName, err)
    } else {
        log.G(context.TODO()).Errorf("Could not add route to IPv6 network %s via device %s: %s", config.AddressIPv6.String(), config.BridgeName, err)
    }
}
```

Reply:
Done.